internal/nexodus/nexodus.go

```diff
@@ -471,10 +471,13 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 	if err != nil {
 		return fmt.Errorf("join error %w", err)
 	}
-
+	deviceRegLogMsg := "Successfully registered device"
+	if modelsDevice.Revision != 0 && modelsDevice.Revision > 1 {
```

Comment:
```suggestion
	if modelsDevice.Revision > 1 {
```

You can drop the first check. It's redundant with the second one.